### PR TITLE
Mark all Tizen.Security.PrivacyPrivilegeManager APIs as deprecated

### PR DESCRIPTION
**Pull request by dmichaluk**
Change-Id: I519abb343b5f4e49d2256a9b0abf69d4fb755632

### Description of Change ###
Mark all Tizen.Security.PrivacyPrivilegeManager APIs as deprecated

### API Changes ###
ACR: https://code.sec.samsung.net/jira/browse/TCSACR-525

### CODE REVIEW

**Comment by shyunMin** [comment]
Looks good to me.

File: src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@ namespace Tizen.Security
     /// The PrivacyPrivilegeManager provides the properties or methods to check and request a permission for privacy privilege.
     /// </summary>
     /// <since_tizen> 4 </since_tizen>
+    [Obsolete("Deprecated in API11, Will be removed in API13. This API will be removed without any alternatives.")]
```
Comment (wiertel) [inline src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs:28]: I see a comma before capitalized word. Please change "API11, Will" -> "API11. Will" OR "API11, will"

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs (hunk 1 above)
Comment (dmichaluk) [reply src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs]: Done.

**Review by wiertel — APPROVED** [review]

**Comment by dmichaluk** [comment]
@wiertel Can you merge this PR?

**Comment by dmichaluk** [comment]
@myroot Can you merge this PR?

**Review by wiertel — APPROVED** [review]

**Comment by myroot** [comment]
> @myroot Can you merge this PR?

In general, merge directly from TG.
I'll do it if you don't have permission.

**Comment by dmichaluk** [comment]
@myroot I don't have permission, "Only those with write access to this repository can merge pull requests."

**Pull request merged**